cf-agent/verify_packages.c

```diff
@@ -201,7 +201,7 @@ PromiseResult VerifyPackagesPromise(EvalContext *ctx, const Promise *pp)
             break;
         case PACKAGE_PROMISE_TYPE_OLD:
             Log(LOG_LEVEL_VERBOSE,
-                "Using old package promise. Please note that this will be obsolete in 3.8");
+                "Using old package promise. Please note that the old implementation is considered obsolete as of 3.8. The old implementation will continue to work, but forward development will be directed toward the new implementation.");
```

Comment:
Very long line, should be split.

Also, don't say that it will continue to work, say that it is deprecated and may be removed in a future version, since that is what we ultimately want to do, and we don't want to give people the impression it will be around forever.